src/core/inlines.js

```diff
@@ -138,6 +151,9 @@ export function run(conf) {
       } else if (t.startsWith("[[")) {
         const nodes = inlineBibrefMatches(t, txt, conf);
         df.append(...nodes);
+      } else if (t.startsWith("|") && t.endsWith("|")) {
```

Comment:
nit: Do we need both checks? The first one should be sufficient.

Reply:
Good catch.